tests/syscall/fs/enc/enc.cpp

```diff
@@ -332,6 +410,8 @@ void test_all(FILE_SYSTEM& fs, const char* tmp_dir)
     cleanup(fs, tmp_dir);
     test_create_file(fs, tmp_dir);
     test_read_file(fs, tmp_dir);
+    test_pread_file(fs, tmp_dir);
```

Comment:
what happens when these tests are run on windows?

Reply:
Like most of the existing tests, it would panic. Note that these tests are limited to Linux for now:
https://github.com/openenclave/openenclave/blob/11090df10b820755eb710b08a473f96d5edb3282/tests/syscall/CMakeLists.txt#L6-L10